tests/v2/validation/envvars/build_versions_test.go

```diff
@@ -0,0 +1,102 @@
+//go:build (infra.any || cluster.any || sanity || validation) && !stress && !extended
+
+package envvars
+
+import (
+	"github.com/rancher/shepherd/clients/rancher"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	steveV1 "github.com/rancher/shepherd/clients/rancher/v1"
+	"github.com/rancher/shepherd/pkg/session"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"testing"
+
+	"github.com/rancher/shepherd/clients/rancher/catalog"
+	"github.com/rancher/shepherd/extensions/charts"
+
+	"github.com/rancher/shepherd/extensions/clusters"
+)
+
+type BuildVersionsTestSuite struct {
+	suite.Suite
+	client               *rancher.Client
+	steveClient          *steveV1.Client
+	session              *session.Session
+	cluster              *management.Cluster
+	clusterName          string
+	chartVersion         string
+	webhookVersion       string
+	cspAdapterMinVersion string
+	fleetVersion         string
+}
+
+func (b *BuildVersionsTestSuite) TearDownSuite() {
+	b.session.Cleanup()
+}
+
+func (b *BuildVersionsTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	b.session = testSession
```

Comment:
Can you change this to: 
`b.session = session.NewSession()`

Reply:
changed, thanks! 😁